CONTRIBUTING.md

```diff
@@ -91,6 +91,10 @@ presented.
    which includes advice for designing APIs for Flutter, and how to
    format code in the framework.
 
+8. [Flutter design doc template](https://docs.google.com/document/d/1SFRO8U2toOlAaZ38dsuEU7Wm5fn41wvBCWKiwADqfmw/edit?usp=sharing),
```

Comment:
please link to this via the flutter.dev/go/template link so that it remains up to date when we change it.